crates/ruff_python_formatter/tests/fixtures.rs

```diff
@@ -140,12 +149,6 @@ fn format() {
                 .unwrap();
             }
         } else {
-            let printed =
-                format_module_source(&content, options.clone()).expect("Formatting to succeed");
-            let formatted = printed.as_code();
-
-            ensure_stability_when_formatting_twice(formatted, options.clone(), input_path);
```

Comment:
Not clear to me why we're doing this, we do this exact thing immediately before the `if`-`else`.